example/lib/post_card/post_card_scenes.dart

```diff
@@ -49,14 +44,22 @@ class PostsListScene extends Scene {
 
   @override
   Widget build() {
-    return MaterialApp(
-      home: Container(
-        color: Colors.grey,
-        padding: const EdgeInsets.symmetric(horizontal: 10),
-        child: ListView.builder(
-          itemCount: posts.length,
-          itemBuilder: (context, index) => PostCard(
-            post: posts[index],
+    return Builder(
+      builder: (context) => MaterialApp(
```

Comment:
what's the builder for?

Reply:
To get `context`. Because `build` needs to be usable from tests (i.e., `tester.pumpWidget(scene.build())`), it doesn't make sense to add the `BuildContext` parameter a `build` function would normally expect. Let me know if I'm missing something though!

Reply:
One thing I found sort of annoying when updating the Scenes to support the environment manipulation panel was the fact that I had to write so much boilerplate for each MaterialApp (`theme`, `darkTheme`, `useInheritedMediaQuery`, etc. etc.). I wonder if I would be useful to introduce some kind of `EnvironmentAwareMaterialApp` to hide some of this complexity? Is there a better way to accomplish this?